test/isolation/specs/insert_dropchunks_race.spec

```diff
@@ -0,0 +1,26 @@
+# Race condition between insert and drop_chunks
+
+setup {
+  CREATE EXTENSION IF NOT EXISTS timescaledb;
+  DROP TABLE IF EXISTS conditions;
```

Comment:
can you name your hypertable something similar to your test, the isolation tests operate on a shared database thats also why you cant rely on the ids being consistant across runs, atm isolationtests are not run in parallel because we only have so few of them but we probably will in the future